### ISSUE TITLE
Bill with multiple payments fully paid and still Outstanding

### ISSUE THREAD

**Issue by mimorita** [nadineproject/nadine#340]
D.W. bill #13616 is closed and paid but still "Outstanding"

**Comment by jsayles** [comment]
The bill is not in the Outstanding Bills page but is in the member's profile and listed as Outstanding.

**Comment by jsayles** [comment]
There were 2 payments on the bill, one for the full amount, and another for $0.  I deleted the $0 bill and it was no longer "outstanding"

**Comment by abkruse** [comment]
How did that $0 payment get applied originally to cause this bug? Is that the bug we should be fixing?

**Comment by mimorita** [comment]
No idea. I don't think I did it (intentionally, that is). I'll keep an eye out for other $0 charges though.

**Comment by jsayles** [comment]
Yes, the $0 payment is it's own bug and we need to see where that came from.  We can enter that if it comes up again.  I've changed the title of this bug to include that multiple payments is the cause of this issue.  

It should be noted that we added a fix in the Outstanding Bills page when this came up previously.  We simply filter out outstanding bills that are actually paid but this might not hold up in the long term when there are many bills in this situation.
